internal/test/e2e/e2e_test.go

```diff
@@ -272,23 +205,30 @@ func installOperator(t *testing.T, ctx context.Context, kubectl *kubectlContext,
 		require.NoError(t, err)
 	}()
 
+	images := []string{*fControllerImage, *fProxyImage, *fRestoreAgentImage, *fBackupAgentImage}
+
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		for _, imageTar := range imageTars {
-			imageFile, err := os.Open(imageTar)
-			require.NoError(t, err)
-			defer func() {
-				assert.NoError(t, imageFile.Close(), "failed to close image tar")
-			}()
-			// Load the built image into the Kind cluster.
-			t.Log("Loading image in to Kind cluster")
-			nodes, err := kind.ListInternalNodes()
-			require.NoError(t, err)
-			for _, node := range nodes {
-				err := node.LoadImageArchive(imageFile)
+		nodes, err := kind.ListInternalNodes()
+		require.NoError(t, err)
+		for _, name := range images {
+			func() {
+				t.Log("Exporting the operator image", name)
+				f, err := ioutil.TempFile("", name)
 				require.NoError(t, err)
-			}
+				defer func() {
+					assert.NoError(t, f.Close(), "failed to close image tar")
+				}()
+				out, err := exec.CommandContext(ctx, "docker", "save", "-o", f.Name(), name).CombinedOutput()
+				require.NoError(t, err, out)
+
+				t.Log("Loading image in to Kind cluster")
+				for _, node := range nodes {
+					err := node.LoadImageArchive(f)
+					require.NoError(t, err)
+				}
+			}()
```

Comment:
Moved the export / import of images to a single function.
This should all be obsoleted by #169